### ISSUE TITLE
Enable ssh service by default

### ISSUE THREAD

**Issue by GorgiAstro** [fayaaz/mixxx-pi-gen#93]
Hi, first thanks a lot for this amazing RPi image!

It looks like the SSH server is not active by default, I had to type `sudo systemctl enable ssh` to enable it. Is it possible to enable the SSH service by default?

**Comment by fayaaz** [comment]
No, I will not be enabling SSH by default - especially when this image comes with a username and password that is the same for all installs.
